scripts/certificates/bash/openssl_ca.cnf

```diff
@@ -104,7 +104,7 @@ extendedKeyUsage = serverAuth
 [ ravendb_custom_server ]
 subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid:always,issuer
-basicConstraints = critical, CA:true, pathlen:0
+basicConstraints = CA:FALSE
```

Comment:
@iftahbe So server is not a CA, but it's still can issue client certificates?

Reply:
The problem is that FireFox rejects the use of a CA cert as a server cert as well.
We don't actually care about the validity of the chain in this regard, since we check the actual cert, so we don't mind that issue.

Reply:
The other problem is that when we generate a client cert from a Let's Encrypt cert, it is not a CA.

Reply:
OK makes sense. I'll retest this later to check if all is fine on docker.